docs/book/stacks-and-components/component-guide/orchestrators/skypilot-vm.md

```diff
@@ -0,0 +1,192 @@
+
+* you want to maximize cost savings by leveraging spot VMs and auto-picking the cheapest VM/zone/region/cloud.
+* you want to ensure high GPU availability by provisioning VMs in all zones/regions/clouds you have access to.
+* you don't need a UI to list all your pipeline runs.
+* you're not willing to maintain Kubernetes based solution or in paying for managed solutions like [Sagemaker](sagemaker.md).
+
+
+## How it works
+
+The SkyPilot VM Orchestrator integrates with the ZenML pipeline framework, allowing you to easily
+incorporate VM provisioning and management into your ML workflows. It abstracts away the complexities
+of managing cloud infrastructure, making it simple to launch jobs and clusters on any cloud provider.
+
+The orchestrator leverages the SkyPilot framework to handle the provisioning and scaling of VMs.
+It automatically manages the process of launching VMs on different cloud providers, ensuring maximum
+cost savings and GPU availability.
+
+One of the key features of the SkyPilot VM Orchestrator is its ability to maximize GPU availability.
+It achieves this by provisioning VMs in all available zones, regions, and clouds, with automatic failover.
+This ensures that your jobs can make the most efficient use of GPU resources.
+
+Additionally, the SkyPilot VM Orchestrator offers cost savings through various mechanisms.
+It leverages managed spot VMs, which can provide significant cost savings compared to on-demand VMs.
+The integration also includes an optimizer that automatically selects the cheapest VM/zone/region/cloud
+for your workloads, further reducing costs. Finally, the orchestrator includes an autostop feature that
+cleans up idle clusters, preventing unnecessary cloud costs.
+
+The SkyPilot VM Orchestrator seamlessly supports your existing GPU, TPU, and CPU workloads, requiring
+no code changes. It provides a unified interface for provisioning and managing VMs across different cloud
+providers, making it easy to run your machine learning workloads in a cost-effective and efficient manner.
+
+{% hint style="warning" %}
+The SkyPilot VM Orchestrator does not currently support the ability to [schedule pipelines runs](https://docs.zenml.io/user-guide/advanced-guide/pipelining-features/schedule-pipeline-runs)
+{% endhint %}
+
+
+## How to deploy it
+
+You don't need to do anything special to deploy the SkyPilot VM Orchestrator. As the SkyPilot integration
+itself takes care of provisioning VMs, you can simply use the orchestrator as you would any other ZenML
+orchestrator. However, you will need to ensure that you have the appropriate permissions to provision VMs
+on your cloud provider of choice and to configure your SkyPilot orchestrator accordingly using the [service
+connectors](https://docs.zenml.io/stacks-and-components/auth-management) feature.
+
+{% hint style="info" %}
+The SkyPilot VM Orchestrator is currently only supported on AWS, GCP, and Azure.
+{% endhint %}
+
+
+## How to use it
```

Comment:
I would make this a tabbed approach for AWS / GCP / Azure and have the How to use it and sevice connector stuff personalized per cloud in this page